### ISSUE TITLE
workbook things I can't fix

### ISSUE THREAD

**Issue by retabak** [bootstrapworld/curriculum#1997]
These tiny issues caught my attention during my workbook review, but I'm not sure how to fix. 

**Measures of Center notes**: ~“When a dataset contains a small number of values, the mode may by the most descriptive measure of center.” It sounds like you’re saying that if you have a SMALL dataset (like, just 5 numbers, say) that mode might be most descriptive. That can’t be; if I wanted to characterize this list - 1, 2, 3, 4, 5 - with just one measure of center, I wouldn’t/ couldn’t choose mode. I think maybe mode works best if there is a lot of repetition in the dataset? (don't know if that is accurate...) We need to be more accurate and specific.~

- [x] On **correlation-is-not-causation** worksheet from Correlations lesson:  “Gender is a confounder: women tend to weigh less and talk more than men.” This felt A LOT like a stereotype (women talk more than men) so I consulted the internet, and the internet agreed. We need a better / different example for this one.

- [x] **which-questions-make-sense-ask** from LR → It took me way too long to figure out why this page has this title. Needs to be more explicit. Can we spell it out? If not on the worksheet, then maybe in the lesson? Is this a theme that we should always consider when we're working with lr-plots?

- [x] **Advanced Displays notes** - "Sometimes machines consume other machines: the stereo from your car could be swapped out for a new one with more features. Each stereo is a machine!" I think we’re maybe trying to say that the car consumes the stereo? Given how we point out that each stereo is a machine, it feels more like the stereo consumes the stereo. That’s not what matters, right? I think we need a better example but I can’t think of one.

**Comment by flannery-denny** [comment]
@retabak 

1. I want to push back on this one. A small number of values is not the same as a small number of data points...
2. agreed. That page could also use more than 3 examples so let's all put our thinking caps on.
3. I've changed the workbook page title and made some lesson adjustments in the commit above. Please check it off if it scratches the itch. (Also opening #2002 because it seems we didn't catch all of the synthesize section rewrites that needed to happen in our slide build work)
4. Would this be clearer for now?

> Sometimes machines consume other machines: For example, a school bus is a machine. It comes with a stereo, which could be swapped out for a new one with more features. Each stereo is a machine. And the bus needs one of them in order to play music over the speakers.

**Comment by flannery-denny** [comment]
@retabak close if you're satisfied

**Comment by retabak** [comment]
Thanks for addressing these! I like the correlation-is-not-causation examples a lot more!

In the advanced display notes, I changed "Each stereo is not a machine" to "A stereo is a machine." When you point out that they are *both* machines, it really feels to me like you're trying to highlight that the stereo consumes the stereo.

To the measures of center notes, I added a note that "a small number of values is not the same as a small number of data points." I think this is a subtle enough distinction that it is valuable to highlight it.
